.github/workflows/ci.yml

```
@@ -11,7 +11,11 @@ on: # yamllint disable-line rule:truthy rule:comments
   pull_request: ~
 
 env:
+<<<<<<< HEAD
```

actionlint (1.7.11)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)